src/components/MultiLineSelect/index.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component, Fragment } from 'react';
+import SelectRows from 'components/SelectRows';
+
+/**
+ * Description
+ * show one or many rows that are removable - top one is always 'addable' in multi-mode
+ * add button triggers onAdd, remove button triggers onRemove
+ */
+
+type MultiLineSelectProps = {
+  selectorObj?: Object,
+  label: string,
+  placeholder: string,
+  viewOnly: boolean,
+  hasError: boolean,
+  errors?: [],
+  multiline: boolean,
+  onChange?: Function,
+  onChangeRaw?: Function,
+  displayData?: null | [{ key: String, value: String }],
+  keyField?: string,
+  baseClassName?: string,
+};
+
+class MultiLineSelect extends Component {
+  constructor(props: MultiLineSelectProps) {
+    super(props);
+    this.state = {
+      selectListVisible: false,
+      selection: props.defaultValue ? [props.defaultValue] : [],
+    };
+  }
+
+  handleChange = rows => {
+    if (this.props.onChangeRaw) {
+      this.props.onChangeRaw(rows);
```

Comment:
we should not need two callbacks... 
i think we can deal with only one onChange that returns the rows
1. rename ...Raw to onChange
2. remove original onChange

Reply:
done